### ISSUE TITLE
Can you provide trained weight file?

### ISSUE THREAD

**Issue by LvRenbaba** [Yunkun-Zhang/CITE#5]
Hi! thanks for your great work. I tried using your code for training, but the accuracy of the training was not as high as you described in the article. can you provide the weight file that you have trained?

**Comment by Yunkun-Zhang** [comment]
Here is our trained weight file using the default config `configs/gastric/cite/clip-vitb16_biolinkbert-l_cb_bs128_i1000.py` and seed 2022: https://sjtueducn-my.sharepoint.com/:u:/g/personal/ykzhang_sjtu_edu_cn/ERg4URWY8_ZDgR0wIwg09dYB-kWkzc7sb15Oi96dphuRyg?e=LkBFVy

If the accuracy of your results deviates significantly from those reported in our paper, please consider the following possibilities:

1. Verify that the data split is correct. You can download the relevant files from issue #1.
2. Adjust the learning rate. Our experiments found that the learning rate has a substantial impact on the results. In the results we reported, with a fixed batch size 64 on 2 GPUs, when `n = 'all'` (all patients), the learning rate was 0.03, whereas when `n = 1`, it was 0.001.
3. Adjust the random seed. We reported the average result of seeds 2021, 2022, and 2023. We also tried some other seeds, but did not observe any significant differences in the results.

**Comment by LvRenbaba** [comment]
it worked.  thank you
